src/redux/messages.js

```diff
@@ -211,7 +211,7 @@ module.exports.getMessages = (username, token, opts) => {
             }
             if (typeof body === 'undefined') {
                 dispatch(module.exports.setStatus('MESSAGE_STATUS', module.exports.Status.MESSAGES_ERROR));
-                dispatch(module.exports.setMessagesError('No session content'));
+                dispatch(module.exports.setMessagesError('getMessages: response had no content'));
```

Comment:
Is this a new style of error message or do we use messages like this elsewhere (e.g. the `getENDPOINT` structure...?

Reply:
I think of the existing style as being structureless -- just a text string that tries to communicate what's going on. In my change, I'm not meaning to impose or propose a new systematic style, just to more accurately and helpfully communicate what's going on, which is that a particular request's response had no content.